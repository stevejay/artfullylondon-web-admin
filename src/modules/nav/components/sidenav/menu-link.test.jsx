import React from 'react'
import { shallow } from 'enzyme'
import _ from 'lodash'

import MenuLink from './menu-link'

it('should render correctly', () => {
  const wrapper = shallow(
    <MenuLink path='/some/path' label='The Label' onClick={_.noop} />
  )

  expect(wrapper).toMatchSnapshot()
})

it('should never update', () => {
  const wrapper = shallow(
    <MenuLink path='/some/path' label='The Label' onClick={_.noop} />
  )

  const result = wrapper.instance().shouldComponentUpdate({})

  expect(result).toEqual(false)
})
