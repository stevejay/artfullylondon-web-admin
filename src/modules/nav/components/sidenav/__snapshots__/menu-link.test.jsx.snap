// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Link
  className="src-modules-nav-components-sidenav-menu-link__link"
  onClick={[Function]}
  replace={false}
  to="/some/path"
>
  The Label
</Link>
`;
